### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
 
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        "George Klooney", "Kianu Reaves",
+        "Scarlett Keegan", "Will Smith",
+        "Jaden Smith", "Scarlett Johansson"
```
Comment (ArsenPidhoretskyi) [inline main.py:13]: Create list with tuples

Comment (ArsenPidhoretskyi) [reply main.py:13]: actors = [("George", "Klooney"), .....]

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (goldenuni) [reply main.py]: Okay, I will do this)

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**